### PR TITLE
fix: remove boilerplate.go.txt

### PR DESCRIPTION
**Pull request by wikkyk**
The copyright status of autogenerated files is at best dubious. The copyright year does not change just because the calendar year has changed. The header will only cause problems every January.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,7 +46,7 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 
 .PHONY: generate
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(CONTROLLER_GEN) object paths="./..."
```
Comment (mcbenjemaa) [inline Makefile:49]: Why not create another special boilerplate go autogenerated.

**Comment by wikkyk** [comment]
depends on #76

File: Makefile (hunk 1 above)
Comment (wikkyk) [reply Makefile:49]: What for?

Comment (mcbenjemaa) [reply Makefile:49]: copyright 😄

Comment (wikkyk) [reply Makefile:49]: Autogenerated files aren't copyrightable.

Comment (mcbenjemaa) [reply Makefile:49]: can you put the boilerplate file empty, and keep this command as it is?

Comment (wikkyk) [reply Makefile:49]: I could but why would we keep an empty file around?

**Review by mcbenjemaa — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (avorima) [reply Makefile:49]: > Autogenerated files aren't copyrightable.

I must disagree here. All files that are compiled to should have the license header even those that are auto-generated. It is not required that the year be kept up to date, so rather the validation that the year matches should be loosened

**Pull request merged**